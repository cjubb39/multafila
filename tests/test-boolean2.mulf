/* more relops in conditional statements, should print hello! cool */

int main(){

	if (1 > 0){
		printOut("gt pass");
	}
	else {
		printOut("gt failed");
	}

	if (0 < 1){
		printOut("lt pass");
	}
	else {
		printOut("lt fail");
	}
	
	if (1 >= 0){
		printOut("geq pass");
	}
	else {
		printOut("geq fail");
	}
	
	if (0 <= 1){
		printOut("leq pass");
	}
	else {
		printOut("leq fail");
	}
	
	return 1;
	
}