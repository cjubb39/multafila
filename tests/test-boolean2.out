
/*==========OUTPUT CODE BELOW==========*/
#include <stdio.h>
#include <pthread.h>
typedef int mf_bool;
#define mf_true 1; 
#define mf_false 0; 
void printOut(char *m){printf("%s\n", m);}
void printInt(int m){printf("%d\n", m);}

int main (  );

pthread_mutex_t global_lock;

pthread_cond_t global_cond;

struct global_lock_struct_s{
} global_lock_struct;

pthread_t global_threads [0];
int main (  )
{

{
pthread_mutex_init( & global_lock , NULL);
pthread_cond_init( & global_cond , NULL);
}
 ;
if(1 > 0 )
{
printOut( "gt pass" ) ;
}

else
{
printOut( "gt failed" ) ;
}
 ;
if(0 < 1 )
{
printOut( "lt pass" ) ;
}

else
{
printOut( "lt fail" ) ;
}
 ;
if(1 >= 0 )
{
printOut( "geq pass" ) ;
}

else
{
printOut( "geq fail" ) ;
}
 ;
if(0 <= 1 )
{
printOut( "leq pass" ) ;
}

else
{
printOut( "leq fail" ) ;
}
 ;
return 1 ;
}

